From 1305ac7e9308dcd59c3acc205bc95097cad87ed5 Mon Sep 17 00:00:00 2001
From: Vladimir Ermakov <vooon341@gmail.com>
Date: Fri, 16 Aug 2013 06:52:26 +0400
Subject: [PATCH 5/5] ARM: dts: wandboard: add binding for wand-rfkill driver

Required gpios pincontrol selected in hog. Add binding only.
Disabled non-removable, because after unblocking need to redetect SDIO device.

Signed-off-by: Vladimir Ermakov <vooon341@gmail.com>
---
 arch/arm/boot/dts/imx6qdl-wandboard.dtsi | 18 +++++++++++++++++-
 1 file changed, 17 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qdl-wandboard.dtsi b/arch/arm/boot/dts/imx6qdl-wandboard.dtsi
index 737805b..c1ef3bd 100644
--- a/arch/arm/boot/dts/imx6qdl-wandboard.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-wandboard.dtsi
@@ -50,6 +50,22 @@
 		spdif-controller = <&spdif>;
 		spdif-out;
 	};
+
+	rfkill {
+		compatible = "wand,imx6qdl-wandboard-rfkill";
+		pinctrl-names = "default";
+		pinctrl-0 = <>;
+
+		bluetooth-on = <&gpio3 13 0>;
+		bluetooth-wake = <&gpio3 14 0>;
+		bluetooth-host-wake = <&gpio3 15 0>;
+
+		wifi-ref-on = <&gpio2 29 0>;
+		wifi-rst-n = <&gpio5 2 0>;
+		wifi-reg-on = <&gpio1 26 0>;
+		wifi-host-wake = <&gpio1 29 0>;
+		wifi-wake = <&gpio1 30 0>;
+	};
 };
 
 &audmux {
@@ -175,7 +191,7 @@
 &usdhc2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usdhc2_2>;
-	non-removable;
+	//non-removable;
 	status = "okay";
 };
 
-- 
1.8.4.rc3

